front/src/modules/project/components/AddOrEditProjectModal.tsx

```diff
@@ -118,8 +118,23 @@ export default function AddOrEditProjectModal({
     }
   };
 
+  const currentProjectNameIsInvalid: boolean =
+    !currentProject.name || currentProject.name.length > 128;
+  const currentProjectObjectivesIsInvalid: boolean =
+    (currentProject.objectives ?? '').length > 4096;
+  const currentProjectDescriptionIsInvalid: boolean =
+    (currentProject.description ?? '').length > 1024;
+  const currentProjectFundersIsInvalid: boolean = (currentProject.funders ?? '').length > 255;
+  const currentProjectBudgetIsInvalid: boolean = (currentProject.budget ?? 0) > 2147483647;
```

Comment:
I think you can make your code more elegant by creating a function checkFields like this (put it in a utils file, have a look in module/Pathfinding to have an example):
```
  const checkFields = (
    project: ProjectForm
  ): {
    name: boolean;
    objectives: boolean;
    description: boolean;
    funders: boolean;
    budget: boolean;
  }
```
Then you can use it directly
```
  const invalidFields = checkFields(currentProject);
```

And you can know if a field is invalid with the function "some"
```
// some returns true as soon as it finds 1 field for which the given callback function is true
const hasError = Object.values(invalidFields).some((fieldIsInvalid) => fieldIsInvalid)
```

Reply:
Done